tools/proofers/char_check.inc

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
We might want to globally rename this from "character check" to "codepoint validator". My worry is that "character check" brings to mind a punctuation checker which this is not.